src/app/coursegrab/models/course.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, **kwargs):
         self.subject_code = kwargs.get("subject_code")
         self.course_num = kwargs.get("course_num")
         self.title = kwargs.get("title")
-        self.search_string = "{} {} {}".format(self.subject_code, self.course_num, self.title)
+        self.search_string = "{}{}{}".format(self.subject_code, self.course_num, self.title.replace(" ", ""))
```

Comment:
i think the python3 standard are f strings:
f"{self.subject_code}{self.course_num}{self.title.replace(' ','')}"